src/components/atoms/VolumeControl/VolumeControl.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useCallback } from "react";
```

Comment:
initial volume control component. Needs to be styles per design https://www.figma.com/file/92gx2sTMvCc2W0NhgAG7pd/SparkleVerse-UI?node-id=1023%3A17756